components/dialog/dialog-mixin.js

```diff
@@ -43,6 +43,7 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 			 */
 			titleText: { type: String, attribute: 'title-text' },
 			_autoSize: { state: true },
+			_focusableContentElem: { state: true },
```

Comment:
Nit: I'd probably name this `_hasFocusableContentElement` or something along those lines. Currently, it sounds like it would be reference to a DOM node. I think we sorta try to avoid prefixes like that though, so it could also be named more along the lines of `_focusContentContainer`. 🤷‍♂️

Reply:
I wondered about that - I also thought _has would be more appropriate but left it out since we avoid those. I'll think a bit more on it.